pkg/loghttp/push/otlp.go

```diff
@@ -118,7 +110,7 @@ func otlpToLokiPushRequest(ld plog.Logs, userID string, tenantsRetention Tenants
 			resAttrs.PutStr(attrServiceName, "unknown_service")
 		}
 		resourceAttributesAsStructuredMetadata := make(push.LabelsAdapter, 0, resAttrs.Len())
-		streamLabels := make(model.LabelSet, len(blessedAttributesNormalized))
+		streamLabels := make(model.LabelSet, 30) // we have a default labels limit of 30 so just initialize the map of same size
```

Comment:
nit: can we get this number (30) from the defaults somehow?

Reply:
It is just an optimization, which would work most of the time, so I think it would be better to keep the code clean and not add an unwanted dependency.